.github/ISSUE_TEMPLATE/breaking-change.yml

```diff
@@ -0,0 +1,107 @@
+name: ".NET breaking change"
+desription: Report a change in .NET that breaks something that worked in a previous version. Intended mostly for product-team use.
+title: "[Breaking change]: "
+labels: breaking-change
+assignees:
+ - gewarren
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Brief description of the breaking change.
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of .NET introduced the breaking change?
+      options:
+        - .NET 6 Preview 6
+        - .NET 6 Preview 7 (Default)
```

Comment:
Will this require regular edits? If so, was it intentional?

Reply:
Yes, and yes. Or they can select "Other" in a pinch.